src/vorta/borg/borg_job.py

```diff
@@ -203,7 +203,13 @@ def prepare(cls, profile):
     @classmethod
     def prepare_bin(cls):
         """Find packaged borg binary. Prefer globally installed."""
-
+        # On MacOS, the PATH environment variable does not seem to be set when run as a pyinstaller binary.
+        # More info at https://github.com/borgbase/vorta/issues/2100
+        # Set the path to also find homebrew installs of Borg, and avoid falling back to the embedded binary.
+        if sys.platform == 'darwin':
+            current_path = os.environ.get("PATH", "")
+            os.environ["PATH"] = f"{current_path}:/opt/homebrew/bin:/usr/local/bin"
```

Comment:
I think this would malfunction if there is not PATH set, resulting in a PATH with a leading column. An empty path element might get interpreted as CWD (iirc) and that is dangerous as it might execute binaries from any path you happen to be in.

Reply:
Added `/usr/bin:/bin` as minimal fallback. The old spec file also had `sbin`, which I think wouldn't be needed since Borg wouldn't be installed as admin tool on macOS.